pages/forbiddenlogin.vue

```diff
@@ -6,45 +6,86 @@ const badIdPType = route.query.type;
-        You have been directed to this page because you have authenticated using
-        an Identity Provider that is based in a country that has been designated
-        a country of concern by the US Government. At this time, we are awaiting
-        clarification from the Department of Justice on the recent executive
-        order on data sharing with countries of concern (<a
-          href="https://www.whitehouse.gov/briefing-room/presidential-actions/2024/02/28/executive-order-on-preventing-access-to-americans-bulk-sensitive-personal-data-and-united-states-government-related-data-by-countries-of-concern/"
-          >Details Here</a
-        >). Please know that we are fully committed to maximizing the impact of
-        the the data from our participants and data sharing but have been
-        advised to for clarification.
-      </p>
+    <div
+      class="font-dark justify-center bg-gradient-to-r from-sky-400 to-emerald-400 bg-clip-text text-5xl font-extrabold text-transparent"
+    >
+      403: Forbidden
     </div>
 
-    <div v-else>
-      <p class="text-2xl font-light leading-normal md:text-3xl">
-        The Identity Provider you have utilized has not been verified by the
-        Data Access committee to provide sufficiently secure user attestation.
+    <div class="flex flex-col items-center">
+      <p class="text-center text-2xl font-light leading-normal md:text-3xl">
+        Thank you for your request.
       </p>
-    </div>
 
-    <p class="md:text-3xl">
-      If you believe you were directed here mistakenly, please contact the
-      <a href="mailto:aireadi-dac@ohsu.edu">Data Access Committee</a>.
-    </p>
-
-    <a href="/">
-      <button
-        class="focus:shadow-outline-blue inline rounded-lg border border-transparent bg-blue-600 px-4 py-2 text-sm font-medium leading-5 text-white shadow transition-colors duration-150 hover:bg-blue-700 focus:outline-none active:bg-blue-600"
+      <div
+        v-if="badIdPType === 'adversarial'"
+        class="container mx-auto justify-center"
       >
-        back to homepage
-      </button>
-    </a>
+        <p class="text-center text-2xl font-light leading-normal md:text-3xl">
+          You have been directed to this page because you have authenticated
+          using an Identity Provider that is based in a country that has been
+          designated a country of concern by the US Government. At this time, we
+          are awaiting clarification from the Department of Justice on the
+          recent executive order on data sharing with countries of concern (<a
+            href="https://www.whitehouse.gov/briefing-room/presidential-actions/2024/02/28/executive-order-on-preventing-access-to-americans-bulk-sensitive-personal-data-and-united-states-government-related-data-by-countries-of-concern/"
+            >Details Here</a
+          >). Please know that we are fully committed to maximizing the impact
```

Comment:
```suggestion
          are awaiting clarification from the Department of Justice on the
          recent <a
            href="https://www.whitehouse.gov/briefing-room/presidential-actions/2024/02/28/executive-order-on-preventing-access-to-americans-bulk-sensitive-personal-data-and-united-states-government-related-data-by-countries-of-concern/"
            >executive order on data sharing with countries of concern</a
          >. Please know that we are fully committed to maximizing the impact
```

Reply:
I don't know how the link renders, but it might be clearer with the link surrounding the details, rather than the separate "Details here" piece.